src/pages/home/HeaderMessage.jsx

```diff
@@ -6,7 +6,14 @@ function HeaderMessage() {
   const breakpoint = 500;
 
   return (
-    <h1 style={styles.message}>
+    <h1
+      className="animate__animated animate__fadeInRight"
+      style={
+      width > 960
+        ? styles.message
+        : styles.messageMobile
+    }
```

Comment:
While you can do this and works to apply different styles at different widths, a better approach would be to use media queries: https://www.w3schools.com/css/css_rwd_mediaqueries.asp. To use media queries in react Im pretty sure you have to have the css in a separate file and import it, you cant inline it. You could also use [React Responsive](https://www.npmjs.com/package/react-responsive) or [Aphrodite](https://www.npmjs.com/package/aphrodite), but I'll let you decide on what you want to use. Media queries also avoid writing redundant styling, it applies on it on top of the current CSS so you only have to specify the fields you want to change.

I would also change this for the other files where you added logic to a style argument